index.bs

```diff
@@ -152,18 +152,18 @@ invoked, the user agent MUST run the following steps:
     |video|, throw a {{InvalidStateError}} and abort these steps.
 4. If the algorithm is not <a>triggered by user activation</a>, throw a
     {{NotAllowedError}} and abort these steps.
-5. Let |pipWindow| be the Picture-in-Picture window that is <dfn>always on
-    top</dfn>. It consistently stays above most other windows.
-6. Let |pipVideo| be the video contained in the |pipWindow|.
-7. If a |pipWindow| is not already created, create one.
-8. Render |video| frames in the |pipVideo|.
-9. Set {{pictureInPictureElement}} to |video|.
-10. <a>Queue a task</a> to <a>fire an event</a> with the name
+5. Set {{pictureInPictureElement}} to |video|.
+6. Let <dfn>Picture-in-Picture window</dfn> be a new instance of
+    {{PictureInPictureWindow}} with its |state| set to |opened|.
```

Comment:
Wouldn't be simpler/shorter to simply instantiate the object and say in PIPWindow spec part that when instantiated, its state is "opened"?